arkouda_metrics_exporter/client/arkouda_metrics_exporter/metrics.py

```diff
@@ -0,0 +1,613 @@
+                timestamp=parser.parse(cast(str, value["timestamp"])),
+                labels=labels,
+            )
+
+        elif scope == MetricScope.USER:
+            user = cast(str, value["user"])
+            labels = [Label("user", value=user)]
+            return UserMetric(
+                name=str(value["name"]),
+                category=MetricCategory(value["category"]),
+                scope=MetricScope(value["scope"]),
+                value=value["value"],
+                timestamp=parser.parse(cast(str, value["timestamp"])),
+                user=user,
+                labels=labels,
+            )
+        else:
+            labels = None
+            return Metric(
+                name=str(value["name"]),
+                category=MetricCategory(value["category"]),
+                scope=MetricScope(value["scope"]),
+                value=value["value"],
+                timestamp=parser.parse(cast(str, value["timestamp"])),
+                labels=labels,
+            )
+
+    def _updateNumberOfRequests(self, metric: Metric) -> None:
+        metricScope = metric.scope
+
+        if metricScope == MetricScope.USER:
+            self._updatePerUserNumberOfRequests(cast(UserMetric, metric))
+        else:
+            self._updateGlobalNumberOfRequests(metric)
+
+    def _updatePerUserNumberOfRequests(self, metric: UserMetric) -> None:
+        metricName = metric.name
+        if metricName == "total":
+            self.perUserTotalNumberOfRequests.labels(
+                user=metric.user, arkouda_server_name=self.arkoudaMetricsServerName
+            ).set(metric.value)
+        else:
+            self.perUserNumberOfRequestsPerCommand.labels(
+                command=metricName,
+                user=metric.user,
+                arkouda_server_name=self.arkoudaMetricsServerName,
+            ).set(metric.value)
+
+    def _updateGlobalNumberOfRequests(self, metric: Metric) -> None:
+        metricName = metric.name
+
+        if metricName == "total":
+            self.totalNumberOfRequests.labels(arkouda_server_name=self.arkoudaMetricsServerName).set(
+                metric.value
+            )
+        else:
+            self.numberOfRequestsPerCommand.labels(
+                command=metricName, arkouda_server_name=self.arkoudaMetricsServerName
+            ).set(metric.value)
+
+    def _updateResponseTimes(self, metric: Metric) -> None:
+        self.responseTimesPerCommand.labels(
+            command=metric.name, arkouda_server_name=self.arkoudaMetricsServerName
+        ).set(metric.value)
+
+    def _updateServerMetrics(self, metric: Metric) -> None:
+        self.numberOfConnections.labels(arkouda_server_name=self.arkoudaMetricsServerName).set(
+            metric.value
+        )
+
+    def _updateReportedTimestamp(self, value: datetime) -> None:
+        self.reportedTimestamp.labels(arkouda_server_name=self.arkoudaMetricsServerName).set(
+            value.timestamp() * 1000
+        )
```

Comment:
I might be missing it but i don't see where this update method is used

Reply:
@pierce314159 good catch! Yeah, that method is no longer needed, will remove

Reply:
@pierce314159 just removed it and retested.

Reply:
Resolving this because I verified it was removed.